openlibrary/macros/LoanStatus.html

```diff
@@ -113,6 +113,7 @@
 
 $if ocaid and secondary_action and availability.get('is_printdisabled'):
   $:macros.BookPreview(ocaid)
+  $:macros.BookSearchInside(ocaid)
```

Comment:
I just wanted to note this if statement feels a bit of a duplication of the one above and that this entire filte could probably do with being simplified.